packages/front-end/styles/global-radix-overrides.scss

```diff
@@ -315,6 +315,10 @@
   }
 }
 
+.rt-Text {
+  margin-bottom: 0 !important;
+}
+
```

Comment:
This seems like a pretty widespread change - is this necessary?

Reply:
+1

Reply:
I think I did this to override the styling coming from the bootstrap stylesheet that messes with the way a few radix components render (i.e. radix tooltip & radix callout), but I can just do the override for the specific components to avoid any unintended widespread changes